src/app/dashboard/component/dashboard.component.ts

```diff
@@ -1,28 +1,90 @@
+import {NgxSpinnerModule, NgxSpinnerService} from 'ngx-spinner';
 
 import 'jquery';
 import 'bootstrap';
-// import {DownloadConfirmationDialogComponent} from '../../download-confirmation-dialog-component/download-confirmation-dialog.component';
 import {FilterService} from '../../services/filter-service';
-import {MatPaginator} from "@angular/material/paginator";
-import {MatTableDataSource} from "@angular/material/table";
+import {MatPaginator, MatPaginatorModule} from '@angular/material/paginator';
+import {
+  MatCell,
+  MatCellDef,
+  MatHeaderCell,
+  MatHeaderCellDef, MatHeaderRow, MatHeaderRowDef, MatRow, MatRowDef,
+  MatTable,
+  MatTableDataSource, MatTableModule
+} from '@angular/material/table';
 import {Subject} from 'rxjs';
 import {debounceTime, distinctUntilChanged} from 'rxjs/operators';
-// import {MatDialog} from "@angular/material/dialog";
+import {FilterComponent} from '../../shared/filter/filter.component';
+import {PhylogenyFilterComponent} from '../../shared/phylogeny-filter/phylogeny-filter.component';
+import {MatExpansionModule, MatExpansionPanel} from '@angular/material/expansion';
+import {MatCheckbox, MatCheckboxModule} from '@angular/material/checkbox';
+import {CommonModule, NgClass, NgStyle} from '@angular/common';
+import {ActiveFilterComponent} from '../../shared/active-filter/active-filter.component';
+import {MatFormField, MatFormFieldModule} from '@angular/material/form-field';
+import {FormsModule} from '@angular/forms';
+import {MatChip, MatChipSet, MatChipsModule} from '@angular/material/chips';
+import {MatButtonModule} from '@angular/material/button';
+import {MatIconModule} from '@angular/material/icon';
+import {MatTooltipModule} from '@angular/material/tooltip';
+import {MatInputModule} from '@angular/material/input';
+import {MatTabsModule} from '@angular/material/tabs';
+import {MatBadgeModule} from '@angular/material/badge';
+import {MatListModule} from '@angular/material/list';
+import {MatDialogModule} from '@angular/material/dialog';
 
 
 @Component({
+  standalone: true,
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
+  imports: [
+    NgxSpinnerModule,
+    FilterComponent,
+    PhylogenyFilterComponent,
+    CommonModule,
+    MatExpansionPanel,
+    MatCheckbox,
+    ActiveFilterComponent,
+    MatFormField,
+    MatTable,
+    MatSort,
+    MatHeaderCell,
+    MatHeaderCellDef,
+    MatCell,
+    MatCellDef,
+    RouterLink,
+    MatChipSet,
+    MatChip,
+    NgStyle,
+    NgClass,
+    MatHeaderRow,
+    MatHeaderRowDef,
+    MatRow,
+    MatRowDef,
+    MatPaginator,
+    FormsModule,
+    MatButtonModule,
+    MatIconModule,
+    MatSortModule,
+    MatTableModule,
+    MatTooltipModule,
+    MatPaginatorModule,
+    MatFormFieldModule,
+    MatInputModule,
+    MatExpansionModule,
+    MatCheckboxModule,
+    MatTabsModule,
+    MatBadgeModule,
+    MatChipsModule,
+    MatListModule,
+    MatDialogModule
+  ],
```

Comment:
Can you check that all these imports are really necessary please, I was able to compile code without any errors with just these imports:
MatPaginatorModule,
    MatTableModule,
    MatInputModule,
    NgxSpinnerModule,
    FilterComponent,
    PhylogenyFilterComponent,
    MatExpansionPanel,
    MatExpansionPanelHeader,
    FormsModule,
    MatCheckboxModule,
    ActiveFilterComponent,
    RouterLink,
    MatChipsModule,
    NgStyle,
    NgClass,
    MatSort